source/4.0/learn/introduction/active-record.html.md

```diff
@@ -1,82 +1,545 @@
 ---
 chapter: Introduction
-title: Active Record and ROM
+title: Compared to ActiveRecord
 ---
 
-This document explains basic differences between Rails' Active Record ORM and ROM.
+$TOC
+  1. [Models vs Relations](#models-vs-relations)
+  2. [Queries](#queries)
+  3. [Associations](#associations)
+  4. [Persistence](#persistence)
+  6. [Validation](#validation)
+  7. [Where ROM Shines](#where-rom-shines)
+$TOC
 
-## Database Support
+Rails ActiveRecord is the most popular persistence
+framework in ruby land. Deployed inside the majority of rails applications
+across the web, it provides API's for quick and simple data access making it a
+great good solution for constructing CRUD style applications.
 
-#### Active Record
+Our intention for this guide is to act as a primer for anyone familiar with
+Rails' ActiveRecord and looking for a quick start guide. Examples in each set will
+show how ActiveRecord accomplishes each task followed by an example
+with the equivalent using ROM.
 
-Supports only SQL.
+All ROM examples are based on `rom-sql` which is an adapter needed to use
+SQL databases with ROM. Information on installing on installing and
+configuring rom-sql for your database can be found in the
+[SQL](/%{version}/learn/sql) guide.
 
-#### ROM
+^
+  Examples below assume a configured environment for each
+  framework. For ROM examples this means an initialized `ROM::Container` with
+  each component registered.
 
-Supports anything that can provide data, including SQL databases, NoSQL
-databases, CSV files, git repositories, remote HTTP APIs, anything. You can work
-with multiple databases at once and combine data in memory, if that's what you
-need.
+  For information on how to configure a ROM environment see either
+  [Setup DSL](/%{version}/learn/getting-started/setup-sql)
+  or
+  [Rails Setup](/%{version}/learn/getting-started/rails-setup)
+  guides.
+^
 
-## Data Access
+^
+  Both frameworks have many similar API's but philosophically they are
+  completely different. In this guide, we attempt to highlight these differences
+  and provide context for why we chose a different path. That is not to say ROM
+  is better than ActiveRecord or vise-versa, it's that they're different and
+  each has its own strengths and weaknesses. 
+^
 
-#### Active Record
 
-Data access logic is part of the object and controls all reading and writing to
-the database. You use the same objects to create, read, update, and delete data.
-These objects are the models in traditional Rails applications.
+## Models vs Relations
 
-#### ROM
+The first difference is ROM doesn't really have a concept of models. ROM objects
+are instantiated by the mappers and have no knowledge about persistence. You can
+map to whatever structure you want and in common use-cases you can use
+repositories to automatically map query results to simple struct-like objects.
```

Comment:
This should be `...use **relations** to automatically...`

Reply:
Whoops, not sure what I was thinking there. Must have been drunk 😄